src/pages/JobList/JobNextRun.js

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment'
+import { string, bool } from 'prop-types'
+
+const JobNextRun = ({ nextExecutionTime, enabled }) => {
+    if (!enabled) {
+        return '-'
+    }
+
+    const now = moment(Date.now())
+
+    /**
+     * The recommendation is to run dhis2 on a server set to UTC time.
+     * In that case this timestamp is also UTC. If those recommendations
+     * weren't followed the time could be off, but there's nothing
+     * we can do to detect that.
+     */
+    const nextRun = moment.utc(nextExecutionTime)
+    const nextRunIsInPast = nextRun.isSameOrBefore(now, 'minute')
+
+    /**
+     * If the time is in the past, that could mean that the task is running,
+     * and the nextExecutionTime hasn't been updated yet.
+     */
+
+    if (nextRunIsInPast) {
+        return 'Not scheduled'
+    }
+
+    /**
+     * According to the moment docs you would expect to() to be the
+     * correct method, as all those examples are describing future events.
+     * But using from() actually results in future relative time,
+     * so that is assumed to be a mistake in the moment docs.
+     */
+    return nextRun.from(now)
```

Comment:
I think this should be `now.to(nextRun)` using `now` as the base date.

Reply:
Ah yes of course. That makes more sense, thx 👍. Fixed in 48f5160